src/non_zero.rs

```diff
@@ -210,6 +210,15 @@ where
     }
 }
 
+impl<T> Default for NonZero<T>
```

Comment:
I'm not sure about this one. Seems a little counter-intuitive. Is it necessary?

Reply:
Deriving Default seems pretty broken to me, since it produces `NonZero(0)`. Having a Default implementation isn't strictly necessary but it does make `NonZero::new(_).map` possible, as used in `Uint::checked_div`.

Reply:
I think it's OK, although I'd note there aren't corresponding `Default` impls on `core::num::NonZero*`